Program.cs

```diff
@@ -1,57 +1,396 @@
+                            response = Console.ReadLine();
+                            response = FormatAnswer(response);
+                            if (!cityStats.ProvinceIsValid(response))
+                            {
+                                Console.Write("\nInvalid response. Please pick from one of the following cities and key in the city name: ");
+                                cityStats.DisplayCitiesList();
+                                Console.WriteLine("Please write your selection from the above list: ");
+                            }
+                        } while (!cityStats.ProvinceIsValid(response));
+                        cityStats.DisplaySmallestPopulationCity(response);
+                        Console.Write("Do you want to return to the Mode Selection Menu? (y/n): ");
+                        if (Console.ReadKey().KeyChar == 'y')
+                            valid = true;
+                        break;
+                    case "4":
+                        Console.Write("\nPlease key in two cities, separated by ',' to compare the population of those two cities: ");
+                        do
+                        {
+                            response = Console.ReadLine();
+                            string[] cities = response.Split(',', 2);
+                            cityOne = cities[0];
+                            cityTwo = cities[1];
+                            cityOne = FormatAnswer(cityOne);
+                            cityTwo = FormatAnswer(cityTwo);
+                            if (!cityStats.CityIsValid(cityOne) || !cityStats.CityIsValid(cityTwo))
+                            {
+                                Console.Write("\nInvalid response. Please pick from two of the following cities and key in the city name: ");
+                                cityStats.DisplayCitiesList();
+                                Console.WriteLine("Please key in two cities, separated by ',' from the above list: ");
+                            }
+
+                        } while (!cityStats.CityIsValid(cityOne) || (!cityStats.CityIsValid(cityTwo)));
+
+                        cityStats.CompareCitiesPopulation(cityOne, cityTwo);
+                        Console.Write("Do you want to return to the Mode Selection Menu? (y/n): ");
+                        if (Console.ReadKey().KeyChar == 'y')
+                            valid = true;
+                        break;
+                    case "5":
+                        Console.Write("\nPlease key in a city's name and a porvinc's name seperated by a comma to display the city on a map(ex. 'London, Ontario'): ");
+                        string cityName;
+                        string provName;
+                        do
+                        {
+                            response = Console.ReadLine();
+                            string[] cityProv = response.Split(',', 2);
+                            cityName = cityProv[0];
```

Comment:
be careful with the same thing I mentioned above, if you're going to get both values from one input you'll need to .Trim() the results to clear the probable space after the comma